src/rtree.ml

```diff
@@ -225,3 +225,11 @@ module Make (E : Envelope) (V : Value with type envelope = E.t) = struct
 end
 
 module Rectangle = Rectangle
+
+(* Iter Function *)
+let rec iter (rtree : Rtree.t) (f : Rtree.tree -> unit) : unit =
+  match rtree with
+  | Rtree.Leaf -> ()  
```

Comment:
You shouldn't need to add the module name here as you are inside the `Rtree` module

```suggestion
  | Leaf -> ()  
```

Reply:
I tried running with Leaf -> () but I encountered some error I had to add an underscore after it, please is that okay?